zeppelin-web-angular/e2e/reporter.coverage.ts

```diff
@@ -0,0 +1,304 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @see https://playwright.dev/docs/test-reporters#custom-reporters
+import { FullResult, Reporter, TestCase, TestResult } from '@playwright/test/reporter';
+import { promises as fs } from 'fs';
+import { flatMap, sortBy } from 'lodash';
+import { scanDirectory, Results } from 'scandirectory';
+import cfg from './reporter.coverage.config';
+
+const TEST_STATUS = {
+  PASSED: 'passed',
+  SKIPPED: 'skipped',
+  FAILED: 'failed'
+} as const;
+
+type ResultsType = Array<[string, number, number, number, number, number]>;
```

Comment:
Could you change this tuple array type into an array of interface with named properties for better readability?
e.g.
```typescript
type Result = {
  path: string;
  total: number;
  success: number;
  failed: number;
  skipped: number;
  rate: number;
}
type Results = Array<Result>
```